root/statistics/Index.js

```diff
@@ -576,22 +579,30 @@ const Index = ({
             <td>{fc('medium.has_discid')}</td>
             <td>{fp('medium.has_discid', 'medium')}</td>
           </tr>
-          {oneToNine.map(num => (
-            <tr key={num}>
-              <th />
-              <th />
-              <th>
-                {texp.ln(
-                  'with {num} disc ID:',
-                  'with {num} disc IDs:',
-                  num,
-                  {num: num},
-                )}
-              </th>
-              <td>{fc('medium.' + num + 'discids')}</td>
-              <td>{fp('medium.' + num + 'discids', 'medium.has_discid')}</td>
-            </tr>
-          ))}
+          {(() => {
+            const discIdRows = [];
+            for (let num = 1; num < 10; num++) {
+              discIdRows.push(
+                <tr key={num}>
+                  <th />
+                  <th />
+                  <th>
+                    {texp.ln(
+                      'with {num} disc ID:',
+                      'with {num} disc IDs:',
+                      num,
+                      {num: num},
+                    )}
+                  </th>
+                  <td>{fc('medium.' + num + 'discids')}</td>
+                  <td>
+                    {fp('medium.' + num + 'discids', 'medium.has_discid')}
+                  </td>
+                </tr>,
+              );
+            }
+            return discIdRows;
+          })()}
```

Comment:
These two could be the same function with an argument `release` or `medium`, couldn't they?

Reply:
Seems like it. Not sure I want to add more refactoring to this PR though since it's big enough as it is. :) Also, this comment is on an outdated diff, since I did end up adding a `mapRange` utility. So no real changes to this file at present.